test/SdkE2ETests/Contents/functions.metadata

```diff
@@ -64,7 +64,13 @@
         "type": "http",
         "direction": "Out"
       }
-    ]
+    ],
+    "retry": {
```

Comment:
Pipeline was failing because of this test - had to add retry information to the JSON file.

Reply:
Will look at the failure, but is the failure caused by new code expecting this to be present? can we enhance that code to make sure it handles scenarios where it isn't?

Reply:
This test runs the SDK's `FunctionMetadataGenerator` against the sample app then compares the generated `functions.metadata` file to this hard copy I've edited. 

The failure was caused by my change in the Sample App where I added RetryAttributes to two of the sample functions as an example but didn't change this file.